### ISSUE TITLE
ENHANCEMENT idea: calling RestController endpoints at rendering time

### ISSUE THREAD

**Issue by icstreispe** [thymeleaf/thymeleaf-spring#238]
In order to simplify and reuse the back-end code you should find a way to call RestControllers endpoints directly from thymeleaf pages/templates at rendering time. This way the back-end could be easily reused for different kind of front-ends. It's a similar approach as with the layouts, when you call other fragments/layouts. It's the jsf approach but simplify their api/implementation in a true spring fashion.
If you don't find my idea ok please share with me why not.

**Comment by danielfernandez** [comment]
I don't think this would make sense. Calling REST controllers from the server side templates would conform an excesively convoluted architecture and break most architectural patterns applicable to the server-side view layer of web applications. Additionally, I don't think it would bring any benefits in terms of reuse.
